Description: Makefile to compile source of the fonts
 According to pkg-fonts policy fonts should be compiled if
 source is available.
Forwarded: no
Author: Vasudeva Kamath <kamathvasudev@gmail.com>
Last-Update: 2011-09-29 14:45
--- /dev/null
+++ b/Makefile
@@ -0,0 +1,9 @@
+default:
+	mkdir TTFfiles
+	./generate.pe devanagari/*.sfd
+	./generate.pe oriya/*.sfd
+	./generate.pe gujarati/*.sfd
+	./generate.pe malayalam/*.sfd
+	./generate.pe tamil/*.sfd
+clean:
+	rm -rf TTFfiles
\ No newline at end of file
